From siccegge@cs.fau.de  Mon May 17 11:13:29 2010
From: Christoph Egger <siccegge@cs.fau.de>
Date: Mon, 17 May 2010 16:57:58 +0200
Subject: driver-core: fix Typo in drivers/base/core.c for CONFIG_MODULE
To: Greg Kroah-Hartman <gregkh@suse.de>, Kay Sievers <kay.sievers@vrfy.org>, Jan Blunck <jblunck@suse.de>, Emese Revfy <re.emese@gmail.com>, "Eric W. Biederman" <ebiederm@aristanetworks.com>, linux-kernel@vger.kernel.org
Cc: vamos@i4.informatik.uni-erlangen.de
Message-ID: <20100517145757.GA3686@faui49.informatik.uni-erlangen.de>
Content-Disposition: inline


In this code section the final S of CONFIG_MODULES was missed making
the whole check useless

Signed-off-by: Christoph Egger <siccegge@cs.fau.de>
Cc: Mark McLoughlin <markmc@redhat.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/base/core.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/base/core.c
+++ b/drivers/base/core.c
@@ -1420,7 +1420,7 @@ struct device *__root_device_register(co
 		return ERR_PTR(err);
 	}
 
-#ifdef CONFIG_MODULE	/* gotta find a "cleaner" way to do this */
+#ifdef CONFIG_MODULES	/* gotta find a "cleaner" way to do this */
 	if (owner) {
 		struct module_kobject *mk = &owner->mkobj;
 
